refactor(curriculum): restructure sudoku solver python project (#54303)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606a3ccb1eea93a23c066bf.md

---
id: 6606a3ccb1eea93a23c066bf
title: Step 14
challengeType: 20
dashedName: step-14
---

# --description--

Now you'll work on a method that finds the empty cells in the sudoku board.

> Within the `Board` class, create an empty method named `find_empty_cell` and give it a `self` parameter.

Reviewer: ```suggestion
Within the `Board` class, create an empty method named `find_empty_cell` that includes a `self` parameter.
```
